Gemfile

```diff
@@ -1,5 +1,7 @@
 source "https://rubygems.org"
 
+gem 'wdm', '>=0.1.0 '
 gem "webrick"
 
 gem "github-pages", group: :jekyll_plugins
+gem "tzinfo-data"
```

Comment:
Hi, thank you for your feedback. I included the tzinfo-data gem in the Gemfile because it is a dependency for the tzinfo gem, which is used for time zone conversions. The tzinfo gem can use either the system zoneinfo files or the Ruby modules from the tzinfo-data gem as the data source. However, on Windows systems, the system zoneinfo files are not available, so the tzinfo-data gem is required. Therefore, I added the tzinfo-data gem to ensure that the application can access the correct time zone information on any platform. I hope this clarifies your question. Also, I was getting errors while running the bundle exec jekyll serve command without the tzinfo-data gem, so I had to include it to make the site work locally.